airflow/providers/cncf/kubernetes/operators/pod.py

```diff
@@ -703,22 +703,21 @@ def execute_complete(self, context: Context, event: dict, **kwargs):
                     remote_pod=pod,
                 )
 
-    def write_logs(self, pod: k8s.V1Pod):
+    def _write_logs(self, pod: k8s.V1Pod):
```

Comment:
is this considered a breaking change?

Reply:
I don't think we need to consider it breaking.  This is a method that realistically no one would ever override.  It's used in one place.  I think of it as "bugfix" that it wasn't marked private.

Really this also connects to something we've discussed before.  That our operators should not be published as "base classes" to be extended.  I think they should be end products whose behavior is governed by semver but not whose _structure_ is governed by semver.  Personally I think we ought to update our "public interface" doc to reflect this.

Reply:
cool yeah I see the discussion in https://github.com/apache/airflow/pull/36767

Reply:
Checked the discussion in https://github.com/apache/airflow/pull/36767, seems about right to me that no one "should" ever override the write_logs method, it is something that should not be changed. Seems like a non breaking change